app/src/store/environmentStore.ts

```diff
@@ -26,13 +28,13 @@ export const useEnvironmentStore = create<State>(set => ({
     })),
 }))
 
-export function environmentFromStr(str: string): Environment | undefined {
+export function environmentFromStr(str: string): Environment {
   switch (str.toLowerCase()) {
     case 'mainnet':
       return Environment.Mainnet
     case 'testnet':
       return Environment.Testnet
     default:
-      return
+      return Environment.Mainnet // default to mainnet
```

Comment:
```suggestion
      return Environment.Mainnet
```

The code already states that clearly 👍